pkg/apis/planetscale/v2/vitesscluster_defaults.go

```diff
@@ -108,3 +122,39 @@ func defaultClusterBackup(backup *ClusterBackupSpec) {
 		backup.Engine = defaultBackupEngine
 	}
 }
+
+func defaultTopoReconcileConfig(t *TopoReconcileConfig) {
+	// Fail fast with static defaults if user didn't supply any field.
+	if t == nil {
+		t = &defaultTopologyReconciliationConfig
+		return
+	}
+
+	// Defaulting registration code.
+	if t.RegisterCells == nil {
+		t.RegisterCells = pointer.BoolPtr(true)
```

Comment:
The default value for each field is now stored in two separate places. We should unify them so there's only one source of truth. Ideally, there would also only be one code path that ever sets the default for a given field.